src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -369,6 +369,12 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     </ResolveTargetingPackAssets>
 
+    <PropertyGroup>
+      <!-- Unescape this semicolon-delimited ordered list so that it's passed as a list to targets
+           that consume it. -->
+      <PackageConflictPreferredPackages>$([MSBuild]::Unescape($(PackageConflictPreferredPackages)))])</PackageConflictPreferredPackages>
```

Comment:
There seems to be an extra `])` at the end.

Reply:
Brace completion will be the death of me. Thanks!

Reply:
I imagine the extra characters might have made the name of the last package unrecognizable, though.

Reply:
Indeed they would have, this was a great catch.